packages/frontend/package.json

```diff
@@ -3,16 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "pnpm generate && next build",
+    "build": "next build",
```

Comment:
does prisma correctly generate types now with just `next build`?

Reply:
Right now the types are generated by the top level `prepare` script on the initial install.

`"prepare": "husky install && pnpm -F db run build && pnpm -F nymjs run build",`

I think we can add the `@personaelabs/db` as a dependency to the `frontend` and have a script to build the `Prisma client` in the `frontend/package.json` if needed. wdyt?